cmd/detectExecuteScan.go

```diff
@@ -520,7 +520,14 @@ func postScanChecksAndReporting(ctx context.Context, config detectExecuteScanOpt
 		}
 	}
 
-	sarif := bd.CreateSarifResultFile(vulns, components)
+	projectVersion, err := sys.Client.GetProjectVersion(config.ProjectName, config.Version)
```

Comment:
Call just GetProjectVersionLink and pass this in subsequent calls as it is called multiple times.

Reply:
Ok, this is good. I see this is cached.